src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+}
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00ACDC;
+}
+
+.product {
+  margin: 50px;
+
+  &__box {
+    max-width: 200px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)
but note the `box-sizing` is the only property allowed to be set directly in * selector